notebooks/tutorials/4_futures_getting_started_lesson3/notebook.ipynb

```diff
@@ -403,20 +403,20 @@
       "        as ticker symbols and resolved relative to the date specified by\n",
       "        symbol_reference_date.\n",
       "\n",
-      "    fields : str or list, optional\n",
+      "    fields : str or list\n",
       "        String or list drawn from {'price', 'open_price', 'high', 'low',\n",
-      "        'close_price', 'volume', 'contract'}. Defaults to 'close_price'.\n",
-      "\n",
-      "    frequency : {'daily', 'minute'}, optional\n",
-      "        Resolution of the data to be returned. Defaults to 'daily'.\n",
+      "        'close_price', 'volume', 'contract'}.\n",
```

Comment:
Default returns OHLCV, right?

Reply:
(Default being no input to `fields`)

Reply:
Actually, it looks like we removed default parameters. I need to update the wording.

Reply:
I just realized that's the docstring xD I thought it was part of the notebook/tutorial text. Nothing to update here.